mvdan/garble
# Add test for literal obfuscators

Test included:
1) Check if the obfuscator is selected randomly.
2) Check that all obfuscators are used
